packages/@webex/plugin-cc/README.md

```
@@ -0,0 +1,70 @@
+- [Testing](#testing)
+- [Samples](#samples) 
+- [Consuming SDK](#consuming-sdk)
+  - [NPM](#npm)
+  - [CDN](#cdn)
+- [Kitchen Sink App](#kitchen-sink-app)
+---
+
+## Getting Started
+The `ContactCenter` package is designed to provide a set of APIs to perform various operations for the Agent flow within Webex Contact Center.
+
+- [Introduction to the Webex Web Calling SDK]()
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)